docker-entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$*" == npm*start* ]]; then
 	if [ ! -e "$GHOST_CONTENT/config.js" ]; then
 		sed -r '
 			s/127\.0\.0\.1/0.0.0.0/g;
-			s!path.join\(__dirname, (.)/content!path.join(process.env.GHOST_CONTENT, \1!g;
+			s!path.join(__dirname, \(.\)/content!path.join(process.env.GHOST_CONTENT, \1!g;
```

Comment:
Oh, I'm sorry that there is no mistake in your script. I ran it for my project today, it has worked very well. Perhaps I don't know well shell. Sorry.